### ISSUE TITLE
Reference base unchanged after liftover from hg38-hg19

### ISSUE THREAD

**Issue by arvindjs** [liguowang/CrossMap#4]
Crossmap Maf does not change the reference base when lifting over from hg38 to hg19. Only genomic coordinates are changed.
Example
NPY4R	5540	BCM	GRCh38	chr10	46462259	46462259	+	Missense_Mutation	SNP	G	G	A

NPY4R	5540	BCM	GRCh37	chr10	47087160	47087160	+	Missense_Mutation	SNP G G A

**Comment by liguowang** [comment]
The reference alleles are indeed updated. "chr10 46462259 46462259" (on hg38) was mapped to the **reverse** strand of "chr10 47087160 47087160" (on hg19). So

chr10 46462259 46462259" (on hg38) == 'G'
chr10 47087160 47087160" (on hg19) == 'C' (revsed strand)
chr10 47087160 47087160" (on hg19) == 'G' (forward strand)

**Comment by arvindjs** [comment]
Yes, Sorry about that, Forgot the strand. thanks for the information!
